pkg/api/server/register_system.go

```diff
@@ -27,6 +27,18 @@ func (s *APIServer) registerSystemHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.Handle(VersionedPath("/libpod/system/prune"), s.APIHandler(libpod.SystemPrune)).Methods(http.MethodPost)
-
+	// Swagger:operation POST /libpod/system/reset libpod resetSystem
+	// ---
+	// tags:
+	//   - system
+	// summary: Reset podman storage
```

Comment:
it resets more than storage no?

Reply:
@sujil02 my point here was that it resets storage, volumes, pods and images, so that should be in the summary.

Reply:
Fixed.